lib/shelley/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -1600,3 +1617,111 @@ matrixNormalTxExamples =
+        describe "no existing key witnesses" $ do
+            describe "updateSealedTx tx noExtraTxBodyContent == Right tx" $ do
+                let testPlutusDir = $(getTestData) </> "plutus"
+                let matrix =
+                        [ "auction_1-2.json"
+                        , "crowdfunding-success-4.json"
+                        , "currency-2.json"
+                        , "escrow-redeem_1-3.json"
+                        , "escrow-redeem_2-4.json"
+                        , "escrow-refund-2.json"
+                        , "future-increase-margin-2.json"
+                        , "future-increase-margin-5.json"
+                        , "future-increase-margin-6.json"
+                        , "future-increase-margin-7.json"
+                        , "future-pay-out-2.json"
+                        , "future-pay-out-5.json"
+                        , "future-pay-out-6.json"
+                        , "future-settle-early-2.json"
+                        , "future-settle-early-5.json"
+                        , "future-settle-early-6.json"
+                        , "game-sm-success-2.json"
+                        , "game-sm-success-4.json"
+                        , "game-sm-success_2-2.json"
+                        , "game-sm-success_2-4.json"
+                        , "game-sm-success_2-6.json"
+                        , "multisig-failure-2.json"
+                        , "multisig-sm-10.json"
+                        , "multisig-sm-11.json"
+                        , "multisig-sm-2.json"
+                        , "multisig-sm-3.json"
+                        , "multisig-sm-4.json"
+                        , "multisig-sm-5.json"
+                        , "multisig-sm-6.json"
+                        , "multisig-sm-7.json"
+                        , "multisig-sm-8.json"
+                        , "multisig-sm-9.json"
+                        , "multisig-success-2.json"
+                        , "ping-pong-2.json"
+                        , "ping-pong-3.json"
+                        , "ping-pong_2-2.json"
+                        --, "prism-3.json" -- Error in $[0]: there should be one 'lovelace' in 'value'
+                        , "pubkey-2.json"
+                        --, "stablecoin_1-2.json" -- Error in $[0]: Value should not be empty
+                        , "stablecoin_1-3.json"
+                        , "stablecoin_1-4.json"
+                        --, "stablecoin_2-2.json" --Error in $[0]: Value should not be empty
+                        , "stablecoin_2-3.json"
+                        , "token-account-2.json"
+                        , "token-account-5.json"
+                        , "uniswap-10.json"
+                        , "uniswap-2.json"
+                        , "uniswap-7.json"
+                        --, "uniswap-9.json" -- Error in $[0]: there should be one 'lovelace' in 'value'
+                        , "vesting-2.json"
+                        ]
```

Comment:
Duplicated with CompatibilitySpec… Could de-duplicate via some shared `TestData` or `Util` module if we want to

Reply:
I know. In the first version I wanted to dump the plutus script. Definitely something could be done to reuse those jsons name as they repeat in 3 places already. Feel free to de-duplicate if you have any idea and is eager to to do it withing this PR

Reply:
Wasn't super-eager, but for the sake of making the property tests neater, I might give it a shot in a follow-up PR.